### PR TITLE
Lmh multiple fixes

### PR DESCRIPTION
**Pull request by LZRS**
- [x] __Update the correct native forms version for [LMH1-73](https://github.com/opensrp/opensrp-client-native-form/pull/608)__

### CODE REVIEW

File: opensrp-chw-core/src/main/res/layout/activity_base.xml [hunk 1]
```diff
@@ -11,435 +10,8 @@
...
-                    android:fontFamily="sans-serif"
-                    android:text="@string/nav_last_sync_time"
-                    android:textColor="@color/white"
-                    android:textSize="14sp" />
-
-            </RelativeLayout>
-        </RelativeLayout>
-
-    </com.google.android.material.navigation.NavigationView>
+    <include layout="@layout/navigation_view" />
```
Comment (paulinembabu) [inline opensrp-chw-core/src/main/res/layout/activity_base.xml:15]: @LZRS Why was this file removed? Doesn't this affect all other chw Flavors?

File: opensrp-chw-core/src/main/res/layout/navigation_view.xml [hunk 2]
```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (paulinembabu) [inline opensrp-chw-core/src/main/res/layout/navigation_view.xml:1]: @LZRS Move this class to CHW and make it just specific to LMH flavor

File: opensrp-chw-core/src/main/res/layout-w900dp/navigation_view.xml [hunk 3]
```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (paulinembabu) [inline opensrp-chw-core/src/main/res/layout-w900dp/navigation_view.xml:1]: @LZRS Move this to CHW and make it Exclusive to LMH flavor

File: opensrp-chw-core/src/main/res/layout-w900dp/activity_base.xml [hunk 4]
```diff
@@ -8,400 +8,6 @@
...
-                    android:fontFamily="sans-serif"
-                    android:text="@string/nav_last"
-                    android:textColor="@color/white"
-                    android:textSize="18sp" />
-
-            </RelativeLayout>
-        </RelativeLayout>
-
-    </com.google.android.material.navigation.NavigationView>
+    <include layout="@layout/navigation_view" />
```
Comment (paulinembabu) [inline opensrp-chw-core/src/main/res/layout-w900dp/activity_base.xml:11]: @LZRS Why was this file piece removed? Doesn't this affect all other chw Flavors?

File: opensrp-chw-core/src/main/res/layout/navigation_view.xml (hunk 2 above)
Comment (LZRS) [reply opensrp-chw-core/src/main/res/layout/navigation_view.xml:1]: I moved them, they are part of this pr https://github.com/opensrp/opensrp-client-chw/pull/1927

File: opensrp-chw-core/src/main/res/layout/activity_base.xml (hunk 1 above)
Comment (LZRS) [reply opensrp-chw-core/src/main/res/layout/activity_base.xml:15]: https://github.com/opensrp/opensrp-client-chw/pull/1927

File: opensrp-chw-core/src/main/res/layout-w900dp/navigation_view.xml (hunk 3 above)
Comment (LZRS) [reply opensrp-chw-core/src/main/res/layout-w900dp/navigation_view.xml:1]: https://github.com/opensrp/opensrp-client-chw/pull/1927

**Review by paulinembabu — APPROVED** [review]
